gammapy/data/event_list.py

```diff
@@ -365,36 +365,48 @@ def select_region(self, regions, wcs=None):
         mask = geom.contains(self.radec)
         return self.select_row_subset(mask)
 
-    def select_parameter(self, parameter, band):
-        """Select events with respect to a specified parameter.
+    def select_parameter(self, parameter, values, is_range=True):
+        """
+        Event selection according to parameter values, either in a range or exact matches.
 
         Parameters
         ----------
-        parameter : str
-            Parameter used for the selection. Must be present in `self.table`.
-        band : tuple or `astropy.units.Quantity`
-            Minimum and maximum value for the parameter to be selected (minimum <= parameter < maximum).
-            If parameter is not dimensionless, a Quantity is required.
+        parameter : `str`
+            Column name to filter on
+            values : `str`, `np.nan`, `tuple`, `list` or `array`
+        Value(s) to match (single value or list) or range bounds (list)
```

Comment:
Maybe by default values should be an iterable. i.e. don't accept str but `[str]`